programs/net-util/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+configure_file(net-util.py net-util.py COPYONLY)
```

Comment:
What is the level of support we'll be providing for this new tool: Is it more like a demo, or a fully supported application? Should it be installed in the .deb package? If so, would the python deps be required or optional? Should there be any documentation anywhere? Should there be some tests for the tool? (maybe there could be a `-n 1` kind of mode like `top`, to make an easy way to validate the output?)

Reply:
@stephenpdeos ^^ My understanding was this was a proof of concept.

Reply:
I agree this is more of a PoC

Reply:
Since a PoC, should it be put somewhere besides `programs/net-util`. Maybe `tools/`?

Reply:
Moved.